analyzers/src/SonarAnalyzer.CSharp/Rules/SymbolicExecution/SymbolicExecutionRunner.cs

```diff
@@ -34,7 +34,7 @@
 using SonarAnalyzer.LiveVariableAnalysis.CSharp;
 using SonarAnalyzer.Rules.SymbolicExecution;
 using SonarAnalyzer.SymbolicExecution;
-using SonarAnalyzer.SymbolicExecution.Roslyn.RuleChecks.CSharp;
+using RuleChecks = SonarAnalyzer.SymbolicExecution.Roslyn.RuleChecks.CSharp;
```

Comment:
The alias can be removed after the old NullPointerDereference is removed from SonarRules.